### DISCUSSION TITLE
Create react play

### DISCUSSION THREAD

**Discussion by ghost** [reactplay/react-play#1117] [General]
Hi! I hope you people are doing well I want to ask if I want to create a new react play do I need to be that particular play assigned to me

**Comment by atapas** [comment]
You just open an issue with the idea of creating the play.

https://github.com/reactplay/react-play/issues/new?assignees=&labels=play-request&template=new-play.yml&title=%5BAdd+a+Play%5D%3A+

**Reply by ghost** [reply]
ok thnx

**Reply by ghost** [reply]
ok I want to create a play that is already in the ideas list so.....

**Reply by atapas** [reply]
You can. You still have to create an issue to create a play.

**Reply by ghost** [reply]
and then that issue will assign to me?
